3_control.md

```diff
@@ -11,6 +11,14 @@ Influencing factors for a stable system:
 How to reconnect a load after an overcurrent event?
 
 Suggestions:
-* Through intelligent communication and negotiation between the things using the energy (the drain) and the source of the energy?
-* By priority? - Drains with small consumption have higher priority? - So that light with higher priority comes before heavy equipment.
-* The source offers/bids free energy capacity by some sort of communication channel. The first drain that requests this free energy, partly or in whole, gets his share. Others follow according to some rules until the free portion of energy is reserved and in used. Drains that don't pick up their request will lose their share og the available energy. ?
+
+1.) Through intelligent communication and negotiation between the things using the energy (the drain) and the source of the energy?
+
+2.) By priority? - Drains with small consumption have higher priority? - So that light with higher priority comes before heavy equipment.
+
+3.) The source offers/bids free energy capacity by some sort of communication channel. The first drain that requests this free energy, partly or in whole, gets his share. Others follow according to some rules until the free portion of energy is reserved and in used. Drains that don't pick up their request will lose their share og the available energy. ?
+
+4.) Letting the output voltage grow liniarly as more current is available. Like with a certain factor between the two. Let's for instance say 12V equals 5 Amp, 24V equals 10 Amp, 36V equals 15 Amp, 48V equals at least 20 Amp - and in between those numbers linearly. On the drain side, there will something knowing this - and according to a priority start consuming. ?
+
+5.) Letting small voltage drops/spikes be signals for consumers to connect - according to some priority as mentioned earlier? So a sort of communication, but via minor voltage changes. Well, actually you can transmit network communication via 230V cables, but it may not be wished to use this requirement for drains, alias consuming units.
```

Comment:
Don't think it is possible to define voltage spikes as signals, as they could easily be confused with noise on the grid.

Power line communication is an option, but also creates lots of issues regarding EMC. As mentioned in the call (around minute 25:00) there is some research going on in this direction to make PLC more cheap and reliable.

But this is related to communication in general, not only to reconnecting of loads after an overcurrent event.

Reply:
I miss the link for that video you reference - who can help with that?...

Reply:
Meeting recordings can be found here: https://open-dc-grid.org/meetings.html